### ISSUE TITLE
Warrior basic charge bug (falling under world cant unstuck).

### ISSUE THREAD

**Issue by Ekstrema** [dalaranwow/dalaran-wow#5211]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

There is really big problem with warriors charging can u fix it pls 2/10 charges i fall under texture (world) and cant unstuck cant do shit. Rly its basic bug and its not fixed.

![wowscrnshot_122717_130230](https://user-images.githubusercontent.com/34887073/34381122-cf60677e-eb06-11e7-8c3e-07b99729a9d7.jpg)

**Comment by Zarjofgt** [comment]
literally unstuck and relog

**Comment by Jiaku** [comment]
/reloadui helps. Every BG you play you fall under the group once or twice. (Charge/Fear/Death coil)

**Comment by Ekstrema** [comment]
Literally i wrote THAT I CANT UNSTUCK AND RELOG
IF u cant fix basic bug like this why to host server?
Truewow was true wotlk server no stupid bugs like this at all everything worked no ITEMS LOOTS LIKE (ITS ALREADY IN USE ) no MOB FOLLOWING U ALL OVER THE WORLD IN COMBAT and more... But guy like Zarjofgt say : literally unstuck and relog (maybe he is one of the unskilled engine guys which cant fix bugs like these thats all. ) Ty for unused bug tracker. Missing truewow.

**Comment by Ekstrema** [comment]
Btw i wanted to reported bug for fix not for asking how to avoid 6 minutes bug skipping ty. Repair it thats all.

**Comment by Revennaras** [comment]
Will take a look what's going there, doing some PTR tests.

- Rev

**Comment by Zarjofgt** [comment]
Ive fallen through the ground like 3 times the past 2 years on this server. Everytime it has been easily fixable with some relogs and shit. Stop flailing on the bug tracker and wasting peoples time.

**Comment by Zarjofgt** [comment]
Also are you implying im a dev? Please tale your autism pills before engaging with the outside world, thank you.

**Comment by Lothloryen** [comment]
Hello there,

It looks like with the recent charge fixes (around 2-3 months ago), there are more chances of falling through textures. The charge is now indeed much more polished and the path calculation/excecution is by far a lot better than it was before, however, as said, this issue might happen (very low chances, though).

So, we will review this and try to perform some improvements.

Thank you for the reports but, try to keep them respectful, please.

**Comment by Ekstrema** [comment]
ZaZarjofgt or what else u are u are stupid person wont argue with u thanks i asked for help to gm not to under 80IQ forest gump but if u are forest u can be rich keep trying to catch fishes. 


Gm thanks for answering but many times when i charge to the hill / mountain (to the higher place i fall under ground) thanks for fixing and happy new year

**Comment by VilePoisons** [comment]
now i know why GM's doesnt asnwers on 90% tickets

**Comment by Lothloryen** [comment]
Hello there,

Several improvements have been done to charge, they are already on the live server.
